physics/src/modules/UniformOcean.cpp

```diff
@@ -0,0 +1,60 @@
+/*!
+ * @file UniformOcean.cpp
+ *
+ * @date 30 Mar 2023
+ * @author Tim Spain <timothy.spain@nersc.no>
+ */
+
+#include "include/UniformOcean.hpp"
+
+#include "include/IFreezingPoint.hpp"
+#include "include/Module.hpp"
+#include "include/constants.hpp"
+
+namespace Nextsim {
+
+void UniformOcean::setData(const ModelState::DataMap& ms)
+{
+    IOceanBoundary::setData(ms);
+    sst = sst0;
+    sss = sss0;
+    mld = mld0;
+    u = u0;
+    v = v0;
+    tf = Module::getImplementation<IFreezingPoint>()(sss[0]);
+    cpml = Water::rho * Water::cp * mld[0];
+    qio = qio0;
+}
+
+UniformOcean& UniformOcean::setSST(double in)
```

Comment:
Not a huge issue, but it can be helpful to have more descriptive variable names than `in`, giving a clue what comes in. This might be a bit redundant in a setter, as you can deduce it from the function name, but personally I'd prefer something like `sst_in` or `sstIn` ( depending what you prefer). I'll add suggestions below, but feel free to ignore them or ask me to change from snake-case to camel-case.

Reply:
```suggestion
UniformOcean& UniformOcean::setSST(double sst_in)
```

Reply:
All done, albeit in CmaleCase, rather than snake_case.